styles/styles.css

```diff
@@ -1265,7 +1265,6 @@ main .section.float-image-right picture {
     font-size: var(--body-font-size-xxl);
   }
 
-  ul,
   ol {
```

Comment:
Sorry for the misunderstanding but we need to keep the ul global style. I meant to delete the the padding on the "global carousel ul style"